src/assets/TOMLAsset.js

```diff
@@ -0,0 +1,21 @@
+const Asset = require('../Asset');
+const toml = require('toml');
+
+class TOMLAsset extends Asset {
+  constructor(name, pkg, options) {
+    super(name, pkg, options);
+    this.type = 'js';
+  }
+
+  parse(code) {
+    return toml.parse(code);
+  }
+
+  generate() {
+    return {
+      js: `module.exports=${JSON.stringify(this.ast, false, 2)};`
```

Comment:
Looks like toml can contain more types than JSON, e.g. dates. Maybe we should use https://github.com/commenthol/serialize-to-js like #682 is going to do for YAML?

Reply:
👍 yeah, probably a good idea. Will update this once it lands.